lib/actions/firebase/firebase.js

```diff
@@ -104,34 +111,39 @@ class FirebaseAction extends Hub.Action {
                                     data: notificationData,
                                 };
                                 try {
-                                    yield this.sendMessageToDevice(deviceId, payload, notificationOptions);
+                                    yield this.sendMessageToDevice(deviceId, webhookId, payload, notificationOptions);
                                 }
                                 catch (error) {
+                                    winston.error(`${LOG_PREFIX} Error in sendMessageToDevice. ${error.toString()} `, { webhookId });
                                     reject(error);
                                 }
                             }
                         }
                     }
                 }
                 catch (error) {
+                    winston.error(`${LOG_PREFIX} Error. ${error.toString()} `, { webhookId });
                     reject(error);
                 }
                 resolve();
             }))
                 .catch((error) => {
+                winston.error(`${LOG_PREFIX} Error. ${error.toString()} `, { webhookId });
                 throw error;
             });
         });
     }
-    sendMessageToDevice(deviceId, payload, options) {
+    sendMessageToDevice(deviceId, webhookId, payload, options) {
         return __awaiter(this, void 0, void 0, function* () {
             return new Promise((resolve, reject) => {
                 FirebaseAction.setFirebaseClient();
                 firebaseAdmin.messaging().sendToDevice(deviceId, payload, options)
-                    .then((_) => {
+                    .then((response) => {
+                    winston.info(`${LOG_PREFIX} notification sent to firebase. ${JSON.stringify(response)}`, { webhookId });
```

Comment:
@phillipperalez : AH++ has info level logging turned on ?

Reply:
or should I move this to warn

Reply:
info is valid and logged

Reply:
👍🏼 
then this PR is ready for review and merge 😄